apps/src/util/i18nStringTracker.js

```diff
@@ -32,9 +33,8 @@ function log(stringKey, source) {
   if (!stringKey || !source) {
     return;
   }
-  if (!window.stringUsageSources[source]) {
-    // Initialize the list of strings for the source if it doesn't already exist.
-    window.stringUsageSources[source] = new Set();
-  }
-  window.stringUsageSources[source].add(stringKey);
+  let url = window.location.origin + window.location.pathname; //strip the query string from the URL
```

Comment:
is there a reason we do this here, rather than in the tracker itself?

Reply:
moved